package/src/custom/tests/School/unit-php/modules/Contacts/Student_GradebookTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Sugarcrm\SugarcrmTestsUnit\TestMockHelper;
+use Sugarcrm\Sugarcrm\Util\Uuid;
+
+require_once 'include/SugarQueue/SugarJobQueue.php';
```

Comment:
Another case of including `SugarJobQueue` that I really don't think should be necessary.